pip/commands/help.py

```diff
@@ -21,11 +21,11 @@ def run(self, options, args):
             return SUCCESS
 
         if cmd_name not in commands_dict:
```

Comment:
The following is just updating the command for the changes in `get_similar_command`